zhaquirks/tuya/ts004f.py

```diff
@@ -132,45 +143,138 @@ class TuyaSmartRemote004FROK(EnchantedDevice):
     }
 
     device_automation_triggers = {
-        (SHORT_PRESS, BUTTON): {COMMAND: COMMAND_TOGGLE, ENDPOINT_ID: 1, CLUSTER_ID: 6},
-        (LONG_RELEASE, BUTTON): {
-            COMMAND: COMMAND_STOP_MOVE_STEP,
+        # Command mode
+        (SHORT_PRESS, BUTTON): {
+            COMMAND: COMMAND_TOGGLE,
             ENDPOINT_ID: 1,
-            CLUSTER_ID: 768,
+            CLUSTER_ID: 6,
         },
+        # 3 seconds
         (LONG_PRESS, BUTTON): {
             COMMAND: COMMAND_MOVE_SATURATION,
             ENDPOINT_ID: 1,
             CLUSTER_ID: 768,
-            PARAMS: {"move_mode": 1},
+        },
+        (LONG_RELEASE, BUTTON): {
+            COMMAND: COMMAND_STOP_MOVE_STEP,
+            ENDPOINT_ID: 1,
+            CLUSTER_ID: 768,
+        },
+        # rotation without pressing contains step_size where you can get ticks count by formula ((step_size - 1) / 12)
+        (COMMAND_STEP, RIGHT): {
+            COMMAND: COMMAND_STEP,
+            ENDPOINT_ID: 1,
+            CLUSTER_ID: 8,
+            PARAMS: {"step_mode": 0},
```

Comment:
Its the same function as the slow, medium and fast so i think the "ground name" shall being the same.
Perhaps `ROTATED_STEP`, `ROTATED_LAZILY`.
Or making the fast as "Worp" or "Fastest" ? (and moving all naming one step up)
More suggestions is very welcome for getting it better.

Reply:
maybe we can use `SPIN`?